src/web/app/components/question-types/question-edit-details-form/question-edit-details-form.component.ts

```diff
@@ -38,4 +38,26 @@ export abstract class QuestionEditDetailsFormComponent<D extends FeedbackQuestio
   triggerModelChangeBatch(obj: Partial<D>): void {
     this.detailsChange.emit({ ...this.model, ...obj });
   }
+
+  onPointsInput(event: KeyboardEvent): void {
+    const { key } = event;
+    const isBackspace = key === 'Backspace';
+    const isDigit = /[0-9]/.test(key);
+    if (!isBackspace && !isDigit) {
+      event.preventDefault();
+    }
+  }
+
+  onPaste(event: ClipboardEvent): void {
+    const { clipboardData } = event;
+    if (clipboardData == null) {
+      return;
+    }
+    const pastedText = clipboardData.getData('text');
+    const isDigit = /^\d+$/.test(pastedText);
+    if (!isDigit) {
+      event.preventDefault();
+    }
```

Comment:
hey @dlimyy, I think it would be good if we add tests for these new functions

Reply:
Hi @cedricongjh, sorry for getting back to you late! I will add the tests for these new functions soon.